test/xsimd_poly_evaluation.cpp

```diff
@@ -0,0 +1,27 @@
+/***************************************************************************
+* Copyright (c) Johan Mabille, Sylvain Corlay, Wolf Vollprecht and         *
+* Martin Renou                                                             *
+* Copyright (c) QuantStack                                                 *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#include "xsimd/xsimd.hpp"
+
+#include "gtest/gtest.h"
+
+#include "xsimd_test_utils.hpp"
+
+TEST(xsimd, poly_evaluation)
+{
+  xsimd::simd_type<float> x1 = xsimd::simd_type<float>(2);
+  auto h1 = xsimd::horner<xsimd::simd_type<float>, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16>(x1);
+  auto e1 = xsimd::estrin<xsimd::simd_type<float>, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16>(x1);
+  EXPECT_FLOAT_EQ(xsimd::hadd(h1), xsimd::hadd(e1));
```

Comment:
You could initialize a `std::vector` with some random data, and then use functions in `https://github.com/xtensor-stack/xsimd/blob/master/test/xsimd_tester.hpp` to load and store your vector. Then you can use [check_almost_equal](https://github.com/xtensor-stack/xsimd/blob/master/test/xsimd_test_utils.hpp#L259) to compare vector results.

Reply:
Okay, followed the example in xsimd_power_test.{cpp,hpp}.